File: LLM-Translations/GPT-4/codenet/Go/Python/s141943854.py
import sys

def check(e):
    if e:
        sys.exit(f"error: {e}")

def main():
    sage = input()

    if sage == "1":
        print("Hello World")
    elif sage == "2":
        total = 0
        for i in range(2):
            try:
                num = int(input())
            except ValueError as e:
                check(e)
            total += num
        print(total)

if __name__ == "__main__":
    main()

File: LLM-Translations/GPT-4/codenet/Go/C/s141943854.c

#include <stdio.h>
#include <stdlib.h>

void check(int e) {
    if (e) {
        fprintf(stderr, "error: %d\n", e);
        exit(1);
    }
}

int main() {
    char sage[10];
    fgets(sage, sizeof(sage), stdin);

    switch (atoi(sage)) {
        case 1:
            printf("Hello World\n");
            break;
        case 2: {
            int sum = 0;
            for (int i = 0; i < 2; i++) {
                int num;
                int result = scanf("%d", &num);
                check(result);
                sum += num;
            }
            printf("%d\n", sum);
            break;
        }
    }

    return 0;
}

// 
